package.json

```diff
@@ -49,14 +49,14 @@
   "scripts": {
     "start": "npm run start:api",
     "start:api": "node src/index.js",
+    "start:handlers": "node src/handlers/index.js h --quotes --bulk_quotes",
     "watch:api": "npx nodemon src/index.js",
-    "start:handlers": "node src/handlers/index.js",
     "dev": "npm run docker:stop && docker-compose -f docker-compose.yml -f docker-compose.dev.yml up --build -d",
     "lint": "npx standard",
     "lint:fix": "npx standard --fix",
     "test": "npm run test:unit",
-    "test:unit": "jest --runInBand --testMatch  '**/test/unit/**/*.test.js' ",
-    "test:coverage": "jest --runInBand --coverage --coverageThreshold='{}' --testMatch '**/test/unit/**/*.test.js'",
```

Comment:
Any reason as to why `--runInBand` was removed?
Sometimes tests fail due to be ran in parallel, not saying it happens here but it's typically a preventive measure so we don't have stupid CI failures.

Reply:
the only reason to make them run faster.

Reply:
iirc mostly random. Yes you're abosulutely correct that it's the test themselves are not properly isolated/reset.

If you don't run into any problems I'm fine with removing it.